### PR TITLE
Change terminology service -> application

### PR DESCRIPTION
**Pull request by oharsta**

### CODE REVIEW

File: client/src/locale/en.js [hunk 1]
```diff
@@ -599,52 +599,52 @@ const en = {
...
+                created: "Created application token {{name}}",
+                updated: "Updated application token {{name}}",
+                reactivated: "Reactivated application token {{name}}",
             }
         },
         scimServices: {
-            searchPlaceHolder: "Search for SCIM services..",
-            noEntities: "There are no service configured with the mock SCIM endpoint",
+            searchPlaceHolder: "Search for SCIM applications..",
+            noEntities: "There are no application configured with the mock SCIM endpoint",
```
Comment (logan-life) [inline client/src/locale/en.js:639]: applications

File: client/src/locale/en.js [hunk 2]
```diff
@@ -916,73 +916,73 @@ const en = {
...
+        infoUriTooltip: "URL of a website with information about the application",
         privacy_policy: "Privacy policy",
-        privacy_policyPlaceholder: "The Private policy URL  of the service",
+        privacy_policyPlaceholder: "The Private policy URL  of the application",
         privacy_policyTooltip: "A Privacy Policy is a legal requirement for all websites and apps that collect or use personal information from users.",
         accepted_user_policy: "Acceptable use policy",
-        accepted_user_policyPlaceholder: "The Acceptable Use Policy (AUP) of the service",
-        accepted_user_policyTooltip: "An acceptable use policy (AUP) is a document stipulating constraints and practices that a user must agree to for access to a service or system.",
+        accepted_user_policyPlaceholder: "The Acceptable Use Policy (AUP) of the application",
+        accepted_user_policyTooltip: "An acceptable use policy (AUP) is a document stipulating constraints and practices that a user must agree to for access to an application or system.",
```
Comment (logan-life) [inline client/src/locale/en.js:983]: let's standardize the capitalization of Privacy Policy and Acceptable Use Policy here (dealer's choice)

File: client/src/locale/en.js [hunk 3]
```diff
@@ -1074,36 +1074,36 @@ const en = {
...
             ldapClient: "LDAP client",
-            ldapEnabledTooltip: "Allow this service to query the SRAM LDAP server",
+            ldapEnabledTooltip: "Allow this application to query the SRAM LDAP server",
             ldapDisclaimer: "LDAP is disabled",
         },
         aup: {
             title: "Reset AUP",
-            confirmation: "Resetting the AUP will require all user of this service '{{name}}' to accept its AUP again. Do you want that?",
-            flash: "Acceptance of the AUP for service {{name}} has been reset for all users"
+            confirmation: "Resetting the AUP will require all user of this application '{{name}}' to accept its AUP again. Do you want that?",
```
Comment (logan-life) [inline client/src/locale/en.js:1090]: users

File: client/src/locale/en.js [hunk 4]
```diff
@@ -1548,7 +1548,7 @@ const en = {
         expiryDate: "Invitation expires on",
         expiryDateTooltip: "The expiration date of the invitation, after which the invitation can no longer be accepted.",
         membershipExpiryDate: "Membership valid until",
-        membershipExpiryDateTooltip: "The expiration date of the membership, after which the user cannot log in to services connected to this collaboration.",
+        membershipExpiryDateTooltip: "The expiration date of the membership, after which the user cannot log in to applications connected to this collaboration.",
```
Comment (logan-life) [inline client/src/locale/en.js:1551]: maybe a good chance to standardize on "cannot" vs "can not". 

Mostly we are using "can not" which is a little awkward to me, but if it is the preferred way that's okay too.

File: client/src/locale/en.js [hunk 5]
```diff
@@ -1586,33 +1586,33 @@ const en = {
...
-        actionsTooltip: "Delete the service Connection request. The request can henceforth not be approved or denied anymore.",
-        serviceConnectionRequests: "Service Connection Requests",
-        serviceConnectionRequestDeleted: "Deleted request to connect service {{service}} to collaboration {{collaboration}}.",
-        serviceConnectionRequestResend: "Resent request to connect service {{service}} to collaboration {{collaboration}}.",
-        serviceRestrictedInfo: "This collaboration is service restricted. Only the platform admin can configure services to be used by this collaboration.",
-        serviceDeleteConfirmation: "Are you sure you want to remove this service from collaboration {{collaboration}}?",
-        serviceConnectionRequestDeleteConfirmation: "Are you sure you want to retract this service connection request?",
-        serviceConnectionRequestResendConfirmation: "Are you sure you want to resend this service connection request?",
+        actionsTooltip: "Delete the application connection request. The request can henceforth not be approved or denied anymore.",
+        serviceConnectionRequests: "Application connection Requests",
```
Comment (logan-life) [inline client/src/locale/en.js:1595]: Application connection requests? Application Connection Requests?

File: client/src/locale/en.js [hunk 6]
```diff
@@ -1586,33 +1586,33 @@ const en = {
...
-        noServices: "This organisation has no services connected.",
-        searchServices: "Search, select and add services to the available services within organisation {{name}}.",
-        deleteServiceTooltip: "Make this service unavailable in the organisation {{name}}. <strong>Note</strong>: the service itself is NOT deleted. It is only not available anymore for the members of the collaborations on this organisation",
-        noAccessAllowed: "This service cannot be enabled for all collaborations, because none collaborations can connect.",
-        notAllowedOrganisation: "This service cannot be enabled for all collaborations, because it does not allow automatic connections. Connections can still be requested by individual collaborations in your organisation.",
-        notEnabledOrganisation: "This service cannot be enabled for all collaborations, because the service does not allow access for this organisation.",
-        serviceRestrictedOrganisation: "This service cannot be made mandatory for all collaborations, because this organisation can only link SURF services.",
+        title: "Applications for application {{name}}",
```
Comment (logan-life) [inline client/src/locale/en.js:1604]: Is this supposed to be 

Applications for organisation {{name}}?

File: client/src/locale/en.js [hunk 7]
```diff
@@ -1586,33 +1586,33 @@ const en = {
...
+        connectedServices: "Connected applications to {{name}}",
+        noServices: "This organisation has no applications connected.",
+        searchServices: "Search, select and add applications to the available applications within organisation {{name}}.",
+        deleteServiceTooltip: "Make this application unavailable in the organisation {{name}}. <strong>Note</strong>: the application itself is NOT deleted. It is only not available anymore for the members of the collaborations on this organisation",
+        noAccessAllowed: "This application cannot be enabled for all collaborations, because none collaborations can connect.",
```
Comment (logan-life) [inline client/src/locale/en.js:1609]: because no collaborations can connect

File: client/src/locale/en.js [hunk 8]
```diff
@@ -1642,7 +1642,7 @@ const en = {
         namePlaceholder: "Name of the group",
         short_name: "Short name",
         shortNamePlaceHolder: "Short name of the group",
-        shortNameTooltip: "A service group's short name will be used (1) for groups provisioned to collaborations, prepended by short name of the service and separated by a hyphen '-' and (2) in the challenge URL generated by PAM web login..<br/><br/>" +
+        shortNameTooltip: "A application group's short name will be used (1) for groups provisioned to collaborations, prepended by short name of the application and separated by a hyphen '-' and (2) in the challenge URL generated by PAM web login..<br/><br/>" +
```
Comment (logan-life) [inline client/src/locale/en.js:1645]: An

Comment (logan-life) [inline client/src/locale/en.js:1645]: extra period

File: client/src/locale/en.js [hunk 9]
```diff
@@ -1741,30 +1741,30 @@ const en = {
...
+                info: "The application acceptable use policy (AUP) has changed. Please read it and indicate below whether you accept.",
+                infoMultiple: "These application acceptable use policies (AUPs) have changed. Please read them and indicate below whether you accept."
             },
-            singleInfo: "A service used by this collaboration requires that you agree to their acceptable use policy.",
-            multipleInfo: "Services used by this collaboration require that you agree to their acceptable use policies.",
-            singleCheck: "I agree to the service's acceptable use policy",
-            multipleCheck: "I agree to the services' acceptable use policies",
-            singleInfoAccepted: "You have already agreed to the acceptable use policy for the service used by this collaboration.",
-            multipleInfoAccepted: "You have already agreed to all of the acceptable use policies for the services used by this collaboration.",
+            singleInfo: "A application used by this collaboration requires that you agree to their acceptable use policy.",
```
Comment (logan-life) [inline client/src/locale/en.js:1762]: An

File: client/src/locale/en.js [hunk 10]
```diff
@@ -1800,11 +1800,11 @@ const en = {
         }
     },
     serviceConnectionRequest: {
```
Comment (logan-life) [inline client/src/locale/en.js:1802]: Application Connection request? Application Connect request?

File: client/src/locale/en.js [hunk 11]
```diff
@@ -1814,9 +1814,9 @@ const en = {
         declineConfirmation: "Are you sure you want to decline this request?",
         deleteConfirmation: "Are you sure you want to delete this request?",
         flash: {
-            declined: "Service Connect request for service {{name}} was declined",
-            accepted: "Service Connect request for service {{name}} was accepted",
-            deleted: "Service Connect request for service {{name}} was deleted",
+            declined: "Application Connect request for application {{name}} was declined",
+            accepted: "Application Connect request for application {{name}} was accepted",
```
Comment (logan-life) [inline client/src/locale/en.js:1818]: Application Connection request? Application connection request?

File: client/src/locale/en.js (hunk 5 above)
Comment (logan-life) [reply client/src/locale/en.js:1595]: Also see above use of "application connect request" 

let's standardize on one

File: client/src/locale/en.js (hunk 1 above)
Comment (logan-life) [inline client/src/locale/en.js:639]:
```suggestion
            noEntities: "There are no applications configured with the mock SCIM endpoint",
```

File: client/src/locale/nl.js [hunk 12]
```diff
@@ -1990,11 +1990,11 @@ const nl = {
         jobName: "Naam",
         jobNextRun: "Geplande run-time",
         runOutdatedRequestsInfo: "Voer de cron-taak uit om alle verouderde open join-verzoeken en nieuwe samenwerkingsverzoeken voor de geconfigureerde drempel te rapporteren",
-        runOpenRequestsInfo: "Run the weekly cron job to report all open requests for CO, Org and Service admins",
+        runOpenRequestsInfo: "Run the weekly cron job to report all open requests for CO, Org and application admins",
```
Comment (logan-life) [inline client/src/locale/nl.js:1993]:
```suggestion
        runOpenRequestsInfo: "Run the weekly cron job to report all open requests for CO, Org and Application admins",
```

File: client/src/locale/nl.js [hunk 13]
```diff
@@ -2050,8 +2050,8 @@ const nl = {
         },
         scim: {
             info: "If enabled there is a mock remote SCIM client available to test the SCIM events." +
-                "Configure a Service with the following SCIM URL to start testing the SCIM functionality",
-            services: "SCIM enabled Services",
+                "Configure a applicatie with the following SCIM URL to start testing the SCIM functionality",
```
Comment (logan-life) [inline client/src/locale/nl.js:2053]:
```suggestion
                "Configure an application with the following SCIM URL to start testing the SCIM functionality",
```

File: client/src/locale/nl.js [hunk 14]
```diff
@@ -2106,8 +2106,8 @@ const nl = {
         orgManager: "organisatiemanager",
         coAdmin: "beheerder",
         coMember: "lid",
-        serviceAdmin: "dienstbeheerder",
-        serviceManager: "dienstmanager",
+        serviceAdmin: "applicatiebeheerder",
+        serviceManager: "applicatiemanager",
```
Comment (logan-life) [inline client/src/locale/nl.js:2110]:
```suggestion
        serviceAdmin: "Applicatiebeheerder",
        serviceManager: "Applicatiemanager",
```

File: server/templates/collaboration_expired_notification.html [hunk 15]
```diff
@@ -17,7 +17,7 @@
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <p>{{ salutation }},</p>
     <p>Please login and change the end date of {{ collaboration.name }} if you want users of this collaboration
-        to login again on the services of this collaboration.</p>
+        to login again on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/collaboration_expired_notification.html:20]:
```suggestion
        to login again to the applications of this collaboration.</p>
```

File: server/templates/collaboration_expired_notification.txt [hunk 16]
```diff
@@ -10,5 +10,5 @@ Click the link below to view this collaboration.
 {{ salutation }},
 
 Please login and change the end date of {{ collaboration.name }} if you want users of this collaboration
-        to login again on the services of this collaboration.
+        to login again on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/collaboration_expired_notification.txt:13]:
```suggestion
        to login again to the applications of this collaboration.
```

File: server/templates/collaboration_expires_warning.html [hunk 17]
```diff
@@ -17,7 +17,7 @@
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <p>{{ salutation }},</p>
     <p>Please login and change the end date of {{ collaboration.name }} to avoid that users of this collaboration
-        can not login anymore on the services of this collaboration.</p>
+        can not login anymore on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/collaboration_expires_warning.html:20]:
```suggestion
        can not login anymore to the applications of this collaboration.</p>
```

File: server/templates/collaboration_expires_warning.txt [hunk 18]
```diff
@@ -10,5 +10,5 @@ Click the link below to view this collaboration.
 {{ salutation }},
 
 Please login and change the end date of {{ collaboration.name }} to avoid that users of this collaboration
-        can not login anymore on the services of this collaboration.
+        can not login anymore on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/collaboration_expires_warning.txt:13]:
```suggestion
        can not login anymore to the applications of this collaboration.
```

File: server/templates/collaboration_suspensed_notification.html [hunk 19]
```diff
@@ -17,7 +17,7 @@
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <p>{{ salutation }},</p>
     <p>Please login and activate {{ collaboration.name }} again if you want users of this collaboration
-        to login again on the services of this collaboration.</p>
+        to login again on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/collaboration_suspensed_notification.html:20]:
```suggestion
        to login again to the applications of this collaboration.</p>
```

File: server/templates/collaboration_suspensed_notification.txt [hunk 20]
```diff
@@ -10,5 +10,5 @@ Click the link below to view this collaboration.
 {{ salutation }},
 
 Please login and activate {{ collaboration.name }} again if you want users of this collaboration
-        to login again on the services of this collaboration.
+        to login again on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/collaboration_suspensed_notification.txt:13]:
```suggestion
        to login again to the applications of this collaboration.
```

File: server/templates/collaboration_suspension_warning.html [hunk 21]
```diff
@@ -17,7 +17,7 @@
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <p>{{ salutation }},</p>
     <p>Please login and activate {{ collaboration.name }} again if you want users of this collaboration
-        to login again on the services of this collaboration.</p>
+        to login again on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/collaboration_suspension_warning.html:20]:
```suggestion
        to login again to the applications of this collaboration.</p>
```

File: server/templates/collaboration_suspension_warning.txt [hunk 22]
```diff
@@ -10,5 +10,5 @@ Click the link below to view this collaboration.
 {{ salutation }},
 
 Please login and activate {{ collaboration.name }} again if you want users of this collaboration
-        to login again on the services of this collaboration.
+        to login again on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/collaboration_suspension_warning.txt:13]:
```suggestion
        to login again to the applications of this collaboration.
```

File: server/templates/membership_expired_notification.html [hunk 23]
```diff
@@ -14,15 +14,15 @@
...
+    <p>As scheduled, your membership of this collaboration has ended. This means you cannot use the applications it
         provides any more. If you think this is a mistake, please contact one of the collaboration's admin, who have
         also received this email.</p>
 </div>
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <h3>Message to the collaboration administrators:</h3>
     <p>Please login and change the suspension date of {{membership.collaboration.name }}
         if you want the user {{ membership.user.name }} of this collaboration to
-        login again on the services of this collaboration.</p>
+        login again on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/membership_expired_notification.html:25]:
```suggestion
        login again to the applications of this collaboration.</p>
```

File: server/templates/membership_expired_notification.txt [hunk 24]
```diff
@@ -13,15 +13,15 @@ You are a member of {{ membership.collaboration.name }}, a collaboration with de
...
+means you cannot use the applications it provides any more. If you think
 this is a mistake, please contact one of the collaboration's admin, who
 have also received this email.
 
 
 Message to the collaboration administrators:
     Please login and change the suspension date of {{membership.collaboration.name }}
     if you want the user {{ membership.user.name }} of this collaboration to
-    login again on the services of this collaboration.
+    login again on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/membership_expired_notification.txt:24]:
```suggestion
    login again to the applications of this collaboration.
```

File: server/templates/membership_expires_warning.html [hunk 25]
```diff
@@ -18,7 +18,7 @@
     <p>{{ salutation }},</p>
     <p>Please login and change the suspension date of {{ membership.collaboration.name }} to avoid that
         the user {{ membership.user.name }} of this collaboration
-        can not login anymore on the services of this collaboration.</p>
+        can not login anymore on the applications of this collaboration.</p>
```
Comment (logan-life) [inline server/templates/membership_expires_warning.html:21]:
```suggestion
        can not login anymore to the applications of this collaboration.</p>
```

File: server/templates/membership_expires_warning.txt [hunk 26]
```diff
@@ -13,6 +13,6 @@ Click the link below to view this membership
 {{ salutation }},
 Please login and change the suspension date of {{ membership.collaboration.name }} to avoid that
 the user {{ membership.user.name }} of this collaboration
-can not login anymore on the services of this collaboration.
+can not login anymore on the applications of this collaboration.
```
Comment (logan-life) [inline server/templates/membership_expires_warning.txt:16]:
```suggestion
can not login anymore to the applications of this collaboration.
```

File: server/templates/service_connection_request.html [hunk 27]
```diff
@@ -1,13 +1,13 @@
 {% extends "mail_layout.html" %}
-{% block title %}SURF service Connection request{% endblock %}
+{% block title %}SURF application Connection request{% endblock %}
```
Comment (logan-life) [inline server/templates/service_connection_request.html:2]:
```suggestion
{% block title %}SURF application connection request{% endblock %}
```

File: server/templates/service_connection_request.txt [hunk 28]
```diff
@@ -1,13 +1,13 @@
 {% extends "mail_layout.txt" %}
-{% block title %}SURF service Connection request{% endblock %}
+{% block title %}SURF application Connection request{% endblock %}
```
Comment (logan-life) [inline server/templates/service_connection_request.txt:2]:
```suggestion
{% block title %}SURF application connection request{% endblock %}
```

File: server/templates/service_connection_request_declined.html [hunk 29]
```diff
@@ -1,23 +1,23 @@
 {% extends "mail_layout.html" %}
-{% block title %}SURF new service connection request declined{% endblock %}
+{% block title %}SURF new application connection request declined{% endblock %}
 {% block content %}
 
 <div class="mail-head" style="background-color: #f5f5f5;padding: 30px 0 20px 60px;">
     <p class="title" style="font-weight: 600;font-size: 38px;line-height: 38px; margin: 0;">
-        Denied connection service request
+        Denied connection application request
```
Comment (logan-life) [inline server/templates/service_connection_request_declined.html:7]:
```suggestion
        Declined connection application request
```

File: server/templates/service_connection_request_declined.html [hunk 30]
```diff
@@ -1,23 +1,23 @@
...
     </p>
     <p class="mail-subtitle" style="font-weight: 300;font-size: 26px;line-height: 30px; margin: 0.6em;">
-        Your connection service request for service {{ service.name }} is denied.</p>
+        Your connection application request for application {{ service.name }} is denied.</p>
```
Comment (logan-life) [inline server/templates/service_connection_request_declined.html:10]:
```suggestion
        Your connection application request for application {{ service.name }} is declined.</p>
```

File: server/templates/service_connection_request_org.html [hunk 31]
```diff
@@ -1,13 +1,13 @@
 {% extends "mail_layout.html" %}
-{% block title %}SURF service Connection request{% endblock %}
+{% block title %}SURF application Connection request{% endblock %}
```
Comment (logan-life) [inline server/templates/service_connection_request_org.html:2]:
```suggestion
{% block title %}SURF application connection request{% endblock %}
```

File: server/templates/service_connection_request_org.txt [hunk 32]
```diff
@@ -1,13 +1,13 @@
 {% extends "mail_layout.txt" %}
-{% block title %}SURF service Connection request{% endblock %}
+{% block title %}SURF application Connection request{% endblock %}
```
Comment (logan-life) [inline server/templates/service_connection_request_org.txt:2]:
```suggestion
{% block title %}SURF application connection request{% endblock %}
```

File: server/templates/service_request_declined.html [hunk 33]
```diff
@@ -1,17 +1,17 @@
 {% extends "mail_layout.html" %}
-{% block title %}SURF service join request declined{% endblock %}
+{% block title %}SURF application join request declined{% endblock %}
 {% block content %}
 
 <div class="mail-head" style="background-color: #f5f5f5;padding: 30px 0 20px 60px;">
     <p class="title" style="font-weight: 600;font-size: 38px;line-height: 38px; margin: 0;">
-        Denied service request
+        Denied application request
```
Comment (logan-life) [inline server/templates/service_request_declined.html:7]:
```suggestion
        Declined application request
```

File: server/templates/service_request_declined.html [hunk 34]
```diff
@@ -1,17 +1,17 @@
...
     </p>
     <p class="mail-subtitle" style="font-weight: 300;font-size: 26px;line-height: 30px; margin: 0.6em;">
-        Your request for service {{ service.name }} is denied.</p>
+        Your request for application {{ service.name }} is denied.</p>
```
Comment (logan-life) [inline server/templates/service_request_declined.html:10]:
```suggestion
        Your request for application {{ service.name }} is declined.</p>
```

File: server/templates/service_request_declined.html [hunk 35]
```diff
@@ -1,17 +1,17 @@
...
 </div>
 <div class="mail-end" style="padding: 30px 0 0 60px;">
     <p>{{ salutation }},</p>
-    <p>Your request to create service {{ service.name }} has
+    <p>Your request to create application {{ service.name }} has
         been declined by the administrator {{ administrator }} of the
         SRAM platform.</p>
     <p>The reason for the denialis:</p>
```
Comment (logan-life) [inline server/templates/service_request_declined.html:17]:
```suggestion
    <p>The reason is:</p>
```

**Review by logan-life — CHANGES_REQUESTED** [review]

File: client/src/locale/nl.js (hunk 14 above)
Comment (FlorisFokkinga) [reply client/src/locale/nl.js:2110]: Original isn't capitalized.

File: client/src/locale/nl.js [hunk 36]
```diff
@@ -599,52 +599,52 @@ const nl = {
...
+                created: "Applicatietoken {{name}} aangemaakt",
+                updated: "Applicatietoken {{name}} bijgewerkt",
+                reactivated: "Applicatietoken {{name}} geheractiveerd",
             }
         },
         scimServices: {
-            searchPlaceHolder: "Search for SCIM services..",
-            noEntities: "There are no service configured with the mock SCIM endpoint",
+            searchPlaceHolder: "Search for SCIM applications..",
+            noEntities: "There are no applications configured with the mock SCIM endpoint",
```
[the remark this answers is not in the document]
Comment (FlorisFokkinga) [reply client/src/locale/nl.js:639]: Yes, but it's only for platform admins.

File: client/src/locale/en.js [hunk 37]
```diff
@@ -2624,7 +2624,7 @@ const en = {
     },
     countDownDialog: {
         title: "Access to {{name}}",
-        subTitle: "To access service {{name}} you will have to be provisioned. This unfortunately takes some time, grab a cup of ☕️.",
+        subTitle: "To access application {{name}} you will have to be provisioned. This unfortunately takes some time, grab a cup of ☕️.",
```
[the remark this answers is not in the document]
Comment (FlorisFokkinga) [reply client/src/locale/en.js:2627]: I don't think it's used.

File: client/src/locale/en.js [hunk 38]
```diff
@@ -2117,41 +2117,41 @@ const en = {
...
-        infoAdmin: "Please be aware that on becoming and admin of this collaboration, your personal data may be shared with the following services. Take a moment to review any policy documents provided by the service.",
-        infoJoinRequest: " Before you can request to join this collaboration, you must review the acceptable use policy (AUP) and privacy policy of the service. Indicate below whether you accept them.",
+        infoMember: "Please be aware that on joining this collaboration, your personal data may be shared with the following applications. Take a moment to review any policy documents provided by the application.",
+        infoAdmin: "Please be aware that on becoming and admin of this collaboration, your personal data may be shared with the following applications. Take a moment to review any policy documents provided by the application.",
+        infoJoinRequest: " Before you can request to join this collaboration, you must review the acceptable use policy (AUP) and privacy policy of the application. Indicate below whether you accept them.",
```
[the remark this answers is not in the document]
Comment (FlorisFokkinga) [reply client/src/locale/en.js:2139]: Dutch has a starting space as well. Not sure if it's there for a reason.

File: client/src/locale/en.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (FlorisFokkinga) [reply client/src/locale/en.js:983]: Done.

**Review by logan-life — COMMENTED** [review]
LGTM

**Pull request merged**